yarn.lock

```diff
@@ -16703,7 +16703,7 @@ shelljs@0.5.3:
   resolved "https://registry.yarnpkg.com/shelljs/-/shelljs-0.5.3.tgz#c54982b996c76ef0c1e6b59fbdc5825f5b713113"
   integrity sha1-xUmCuZbHbvDB5rWfvcWCX1txMRM=
 
-shelljs@^0.8.2:
+shelljs@^0.8.1, shelljs@^0.8.2:
```

Comment:
Should we use this opportunity to align these two deps by upgrading this package to use `0.8.2`? It *shouldn't* be a breaking change in `shelljs`.

Reply:
@emizzle Updated so we use 0.8.x for all packages